### PR TITLE
BUG: Fix RigidAlignmentModuleMetadata

### PR DESCRIPTION
**Pull request by Connor-Bowley**
The metadata was no longer being installed to the correct path during
build. I believe this was because of the relative path that moved up
a directory.

### CODE REVIEW

**Comment by Connor-Bowley** [comment]
@jcfr 
@allemangD 
@bpaniagua 
Please review.

**Review by bpaniagua — APPROVED** [review]
lgtm

File: Modules/Scripted/RigidAlignmentModule/CMakeLists.txt [hunk 1]
```diff
@@ -6,25 +6,21 @@ set(${MODULE_NAME}_TITLE "Rigid Alignment Module" CACHE STRING "Module title")
 set(${MODULE_NAME}_CATEGORY "Groups" CACHE STRING "Module category")
 
 set(MODULE_METADATA_SCRIPT
-  ${CMAKE_CURRENT_BINARY_DIR}/../${MODULE_NAME}Metadata/__init__.py
+  ${CMAKE_CURRENT_BINARY_DIR}/_${MODULE_NAME}Metadata/__init__.py
```
Comment (jcfr) [inline Modules/Scripted/RigidAlignmentModule/CMakeLists.txt:9]: Is the addition of the extra `_` intentional ?

Comment (jcfr) [reply Modules/Scripted/RigidAlignmentModule/CMakeLists.txt:9]: IF yes, consider providing some details in the commit message to justify the rename.

Comment (Connor-Bowley) [reply Modules/Scripted/RigidAlignmentModule/CMakeLists.txt:9]: Yes, it is playing on the `_` means private in Python from [PEP-8](https://peps.python.org/pep-0008/#descriptive-naming-styles). The python module is essentially private to the module.

I will say as such in the commit message.

**Comment by jcfr** [comment]
Could you confirm that (1) this fixes a regression introduced by https://github.com/NIRALUser/GROUPS/commit/38274393811c0d4d3a8d0fb47b0789704fee34d7 and (2) packaging of the extension still works.

**Comment by Connor-Bowley** [comment]
I was able to successfully package on Linux. In the package The `_RigidAlignmentModuleMetadata` folder contained a correct `__init__.py` and was in the `lib/Slicer-5.1/qt-scripted-modules/` directory, which I believe is correct.

**Comment by Connor-Bowley** [comment]
Regarding the regression, is being able to package, as well as having this working successful in a SlicerSALT fork enough to say the regression is no longer an issue?

**Comment by jcfr** [comment]
> Regarding the regression, is being able to package, as well as having this working successful in a SlicerSALT fork enough to say the regression is no longer an issue?

Having it work from both the build and install trees on windows and linux would be sufficient.

I will trigger a windows build & package of the extension to confirm

**Comment by jcfr** [comment]
### Window

#### build

image

#### package

image

**Review by jcfr — APPROVED** [review]

**Comment by Connor-Bowley** [comment]
@jcfr Can you merge? I do not have merge access to this repository.

**Pull request merged**